vertx-web/src/main/java/io/vertx/ext/web/RoutingContext.java

```diff
@@ -613,7 +613,19 @@ default LanguageHeader preferredLanguage() {
   }
 
   /**
-   * Returns a map of named parameters as defined in path declaration with their actual values
+   * Add a new one or replace an existing path parameter
+   * @throws NullPointerException when the name or value is null
+   */
+  void addOrReplacePathParam(String name, String value);
+
+  /**
+   * Remove a path parameter
+   * @return {@code true} when removed, {@code false} otherwise
+   */
+  boolean removePathParam(String s);
+
+  /**
+   * Returns an unmodifiable map of named parameters as defined in path declaration with their actual values
```

Comment:
This is a breaking change that is not mandatory (even if I understand the intention). Can you please revert it?

Reply:
Do you mean that you prefer to expose it directly as it was? without being not modifiable?

Reply:
Yes, because we some users will expect this to be modifiable.